apps/codelab/src/app/components/exercise/exercise.component.css

```diff
@@ -17,6 +17,12 @@
   margin-top: 16px;
 }
 
+.solved-message h3:hover {
```

Comment:
Now that it's clickable can we make it  accessible?
E.g. make it a `button`?

Reply:
Sure. I thought about it.

Reply:
Done.